webpack/base.config.js

```diff
@@ -0,0 +1,39 @@
+import webpack from 'webpack';
+import yargs from 'yargs';
+
+export const argv = yargs
```

Comment:
If we are going to export this how about a better name like `options`, `settings`, or `customizations`?